api/src/main/resources/emails/welcomeemail/content.html

```diff
@@ -3,261 +3,325 @@
+          <div style="font-weight: 400; padding-bottom: 0.6rem; padding-top: 0.2rem">Additional Data Access</div>
+          <div class="stepCard step_rt_ct">
+            <ul style="padding-bottom: 1rem">
+              ${CT_STEPS}
+            </ul>
           </div>
         </div>
-      </td>
-    </tr>
-    <tr style="height: 1em"></tr>
-    <tr class="card">
-      <td colspan="3">
-        <div style="display: flex">
-          <div class="bullet">
-            <img src="${BULLET_3}">
+        <div style="padding-bottom: 1.6rem">
+          <div class="text bold-user-label" style="margin-bottom: 0.1rem">Step 3</div>
+          <div style="font-weight: 400; padding-bottom: 0.6rem; padding-top: 0.2rem">Sign the code of conduct</div>
+          <div class="stepCard step3" style="padding-left: 40px;">
+            <div style="padding-top: 0.5rem">Sign Data User Code of Conduct</div>
           </div>
-          <div class="text">
-            <span>
-              Get familiar with our tools and data by exploring support materials in our ‘User Support Hub’ and example analyses under ‘Featured workspaces’.
-               These resources can be accessed from the left side panel in the Workbench.
+        </div>
+      </div>
+    </td>
+  </tr>
+  <tr>
+    <td colspan="3">
+      <!--  <div class="text">
+         Once you complete Steps 1 and 3, you will have access to Registered Tier data, which includes individual data, EHR data, survey responses, physical measurements and wearable device data. Upon completing Step 2, which is optional, you will gain access to Controlled Tier data, which includes genomic data. Regardless of if you complete Step 2, you must complete Step 3 and sign the Data User Code of Conduct to start using the Researcher Workbench.
+       </div> -->
+    </td>
+  </tr>
+  <tr>
+    <td colspan="3">
+      <div class="text secondTitle">About your Researcher Workbench account</div>
+      <div class="text"> Your Researcher Workbench account is hosted by Google. This is
+        different and isolated from your personal and
+        professional Google accounts. There are two places during the registration process where
+        you will interface directly with Google:
+      </div>
+
+      <div class="text">
+        <label class="bold-user-label">Google Terms of Service agreement:</label>
+        Because the workbench account is hosted by Google,
+        you will be asked to agree to Google’s Terms of Service agreement.
+      </div>
+      <div class="text">
+        <label class="bold-user-label">2-step verification:</label>
+        Setting up 2 step verification on your Workbench account adds an
+        extra level of security to your account.
+      </div>
+
+    </td>
+  </tr>
+  <tr>
+    <td colspan="3">
+      <div class="text">Helpful tips before you get started:</div>
+    </td>
+  </tr>
+  <tr class="card">
+    <td colspan="3">
+      <div style="display: flex">
```

Comment:
oh thanks for pointing it out. I'll change tht

Reply:
Ok so flex is some thing we were using in the existing emails as well. So i am going to leave it as is because of launch release priority, and create a new PR to change that